src/behat/CentreonContext.php

```diff
@@ -464,42 +501,45 @@ public function getStorageDatabase()
     }
 
     /**
-     *  Launch Centreon Web container and setup context.
+     * Launch Centreon Web container and setup context.
      *
-     * @param $name Entry name.
+     * @param string $composeBehatProperty Bind property to docker-compose.yml path
+     * @param string[] $profiles docker-compose profiles to activate
      * @throws \Exception
      */
-    public function launchCentreonWebContainer($name)
+    public function launchCentreonWebContainer(string $composeBehatProperty, array $profiles = []): void
     {
-        $composeFile = $this->getContainerComposeFile($name);
-        if (empty($composeFile)) {
-            throw new \Exception(
-                'Could not launch containers without Docker Compose file for ' . $name . ': '
-                . 'check the configuration of your ContainerExtension in behat.yml.'
-            );
+        foreach ($profiles as $profile) {
+            if (preg_match('/^web(?!driver)/', $profile)) {
+                $this->webService = $profile;
+            }
+        }
```

Comment:
if `$profiles` is an empty array, the remaning code will fail.
Should we manage this case ?

Reply:
Why remaining code will fail ?
`$this->webService` is `web` by default

Reply:
My bad, I missed that. 
That's 🆗 ! 
_"Resolved"_ 👍🏻